src/Employer/Employer.Web/Configuration/ConfigurationExtensions.cs

```diff
@@ -91,7 +89,11 @@ public static void AddMvcService(this IServiceCollection services, IHostingEnvir
 
                 opts.AddTrimModelBinderProvider(loggerFactory);
             })
-            .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<Startup>())
+                //not required for now.
+                //.AddFluentValidation(fv =>
```

Comment:
Can you remove this. We don't want commented code. It is either we need it or we don't.